src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int numberOfBall = 3;
```

Comment:
constants should be in uppercase
```suggestion
    private static final int NUMBER_OF_BALLS = 3;
```